### PR TITLE
Add documentation on how to run werable devices

### PR DESCRIPTION
**Pull request by lrapetti**
With this PR I am adding some documentation on how to run some wearable devices.
This documentation is an updated version of the older documentation contained in https://github.com/robotology/human-dynamics-estimation/wiki/Human-Dynamics-Estimation-IWear-Source-Devices, in this way we also separate the documentation for the two repo and we can remove old documentation.

@Yeshasvitvs @diegoferigo

### CODE REVIEW

File: doc/How-to-run-XsensSuit.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+**Before running**
+Before running the device make sure that:
+- [`yarpserver`](https://www.yarp.it/yarpserver.html) is running
+- The router/receiver of the suit is connected to the laptop, the suit is powered on, and the pendrive with the licence is instered on the laptop.
+- There is not another device running that is using the port names. In case multiple suits have to run simultaneously, it is necessary to differentiate the name of the ports.
+- In the [configuration file](https://github.com/robotology/wearables/blob/master/app/xml/XsensSuitWearableDevice.xml), the following parameters are set propery:
+  - `xsens-rundeps-dir`: Folder where XsensMVN runtime dependencies are stored.
+  - `default-calibration-type`: Calibration type to be used (available values: `NposeWalk`, `TposeWalk`, `Npose`, `Tpose`).
+  - `minimum-calibration-quality-required`: Minimum calibration quality to be considered good enought to be applied and allow the acquisition to start (available values: `Poor`, `Acceptable`, `Good`).
+  - `body-dimensions`: Subject=specific body dimensions.
```
Comment (diegoferigo) [inline doc/How-to-run-XsensSuit.md]: Typo Subject=specific

File: doc/How-to-run-XsensSuit.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- The subject is wearing the suit correctly 
+
+**Calibration**
+In order to acquire data from the Xsens suit it is required to perform a calibration procedure, the type of procedure is set trought the configuration file (`default-calibration-type`).
```
Comment (diegoferigo) [inline doc/How-to-run-XsensSuit.md]: Typo trought

File: doc/How-to-run-XsensSuit.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+**Data**
+Once the calibration procedure is completed with the required minimum quality level, the data acquisition is started sending the following command:
+```
+yarp rpc /XsensSuit/Control/rpc:i
+startAcquisition
+```
+If the device is running correctly, the stream of wearable data can be read with:
+```
+/XsensSuit/WearableData/data:o
```
Comment (diegoferigo) [inline doc/How-to-run-XsensSuit.md]: Missing `yarp read ...`?

File: doc/How-to-run-XsensSuit.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+```
```
Comment (diegoferigo) [inline doc/How-to-run-XsensSuit.md]: I don't remember, do we have also a `stopAcquisition` RPC command?

Comment (lrapetti) [reply doc/How-to-run-XsensSuit.md]: yes, we do

Comment (diegoferigo) [reply doc/How-to-run-XsensSuit.md]: :+1:

File: doc/How-to-run-XsensSuit.md (hunk 1 above)
Comment (lrapetti) [reply doc/How-to-run-XsensSuit.md]: fixed https://github.com/robotology/wearables/pull/60/commits/b04212af1f80f784c4176662c1fffae0dda95c32

File: doc/How-to-run-XsensSuit.md (hunk 2 above)
Comment (lrapetti) [reply doc/How-to-run-XsensSuit.md]: fixed https://github.com/robotology/wearables/pull/60/commits/b04212af1f80f784c4176662c1fffae0dda95c32

File: doc/How-to-run-XsensSuit.md (hunk 3 above)
Comment (lrapetti) [reply doc/How-to-run-XsensSuit.md]: fixed https://github.com/robotology/wearables/pull/60/commits/b04212af1f80f784c4176662c1fffae0dda95c32

**Pull request merged**